jest/config.js

```diff
@@ -0,0 +1,16 @@
+const { defaults } = require('jest-config');
+
+module.exports = {
+  rootDir: '../',
+  testMatch: ['<rootDir>/src/**/*.test.{ts,tsx}'],
+  testPathIgnorePatterns: ['/dist/', '/.github/', '/node_modules/'],
+  moduleFileExtensions: [...defaults.moduleFileExtensions, 'ts', 'tsx'],
+  setupFilesAfterEnv: ['<rootDir>/jest/setup.ts'],
+  verbose: true,
+  transform: {
```

Comment:
I think that we normally wouldn't want this explicit transform. By default jest picks up the top level babel config, so if we re-use it then we wouldn't need this manual transform 😃